inc/mlx90632.h

```diff
@@ -276,11 +290,23 @@ double mlx90632_calc_temp_object_reflected(int32_t object, int32_t ambient, doub
  * @note EEPROM version can have swapped high and low bytes due to CPU or I2C.
  * Please confirm that i2c read (16bit) is functioning as expected.
  *
- * @retval 0 Successfully initialized MLX90632 driver
+ * @retval 0 Successfully initialized MLX90632 driver, extended range measurement not supported
+ * @retval @link ERANGE @endlink Successfully initialized MLX90632 driver, extended range measurement is supported
  * @retval <0 Something went wrong. Consult errno.h for more details.
  */
 int32_t mlx90632_init(void);
 
+/** Trigger start measurement for mlx90632
+ *
+ * Trigger measurement cycle and wait for data to be ready. It does not read anything, just triggers and completes.
+ *
+ * @retval <0 Something failed. Check errno.h for more information
+ * @retval >=0 Channel position where new (recently updated) measurement can be found
+ *
+ * @note This function is using usleep so it is blocking!
+ */
+int mlx90632_start_measurement(void);
```

Comment:
Line 325 should be removed as it is double declaration. Kinda strange none of the compilers complain